main.py

```diff
@@ -2,6 +2,31 @@
 
 from django.db.models import QuerySet
 
+from db.models import Actor
+from db.models import Genre
+
 
 def main() -> QuerySet:
-    pass
+    genres = ["Western", "Action", "Dramma"]
+    actors = ["George Klooney", "Kianu Reaves", "Scarlett Keegan", "Will Smith", "Jaden Smith", "Scarlett Johansson"]
+    for genre in genres:
+        Genre.objects.create(name=genre)
+
+    for actor in actors:
+        first_name, last_name = actor.split(" ")
+        Actor.objects.create(first_name=first_name, last_name=last_name)
+
+    Genre.objects.filter(name="Dramma").update(name="Drama")
+    Actor.objects.filter(last_name="Klooney").update(last_name="Clooney")
+    Actor.objects.filter(first_name="Kianu").update(first_name="Keanu", last_name="Reeves")
+
+    action_genre = Genre.objects.filter(name="Action")
+    if action_genre is not None:
+        action_genre.delete()
+
+    actress_scarlett = Actor.objects.filter(first_name="Scarlett")
+    if actress_scarlett is not None:
+        actress_scarlett.delete()
```

Comment:
Similar to the previous comment, `actress_scarlett` is a queryset and will never be `None`. Use `if actress_scarlett.exists():` to check if the queryset contains any records before attempting to delete.